src/locale/Language.js

```diff
@@ -55,12 +55,28 @@ export default class Language {
   }
 
   getMonthByAbbrName(name) {
-    const monthValue = this._monthsAbbr.findIndex((month) => month === name) + 1
+    let index = -1
+    this._monthsAbbr.some((month, i) => {
```

Comment:
Whats the improvment? The left side has the same outcome and is a lot easier to read

Reply:
I agree with you, but I didn't make any changes to this file. I just took the code that was [already here](https://github.com/sumcumo/vue-datepicker/blob/master/src/locale/Language.js)